projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -30,6 +30,58 @@ <h2 class="row no-gutters section-header-heading">
               }}
             </div>
           </h2>
+          <ng-container *ngIf="oboCustomers$ | async as oboCustomers">
+            <ng-container *ngIf="checkActiveOboCustomers(oboCustomers.entries)">
```

Comment:
I would suggest we try to refactor this, since it's not a good practice to evaluate methods in templates.